### PR TITLE
gRPC - App logs

### PR DESCRIPTION
**Pull request by drgarcia1986**
* Implement app logs
* add pod name in front of log msg
* add gRPC Stream Interceptor
* add K8S Client based on `client-go`
* Remove go-swagger implementation
* Clean up

(related #203 )

---
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa-api/205)

### CODE REVIEW

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -849,6 +849,611 @@
...
+			"Rev": "e121606b0d09b2e1c467183ee46217fa85a6b672"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/transport",
+			"Comment": "v2.0.0",
+			"Rev": "e121606b0d09b2e1c467183ee46217fa85a6b672"
+		},
+		{
 			"ImportPath": "k8s.io/kubernetes/pkg/api",
 			"Comment": "v1.4.0-alpha.2-550-gda53a24",
```
Comment (aguerra) [inline Godeps/Godeps.json]: Not sure if these `k8s.io/apimachinery` shoud be here.

Comment (drgarcia1986) [inline Godeps/Godeps.json]: I have tried to remove from Godeps.json and run `godeps save ./...`, but it's back again :/

**Comment by drgarcia1986** [comment]
@aguerra I don't understand, are you suggest to remove arg `podName` for `app.K8sOperations.PodLogs()` interface? I don't follow, because, we need to get logs for a single pod (e.g. deploy). About pod status, can you explain your idea ?

**Comment by aguerra** [comment]
As we talked in person this way is more granular so ok4me.

File: pkg/server/app/app.go [hunk 2]
```diff
@@ -21,28 +33,90 @@ type AppOperations struct {
...
+		if t.Name == team {
 			found = true
 			break
 		}
 	}
 	return found
 }
 
+func (ops *AppOperations) getAppTeam(appName string) (string, error) {
+	annotation, err := ops.kops.NamespaceAnnotation(appName, TeresaAnnotation)
```
Comment (drgarcia1986) [inline pkg/server/app/app.go]: Change to `Label` instead of `Annotation`

**Pull request merged**